### ISSUE TITLE
Out of date with Three.js r166

### ISSUE THREAD

**Issue by kitaedesigns** [ektogamat/lensflare-threejs-vanilla#3]
Not sure exactly why but I believe this repo is broken with newer versions of three js. Tried changing the three js version to 0.166.0 in package.json and rebuilding and now it's black. 
![image](https://github.com/user-attachments/assets/cd235517-040d-4442-a6bb-c45c9ac37cef)
![image](https://github.com/user-attachments/assets/0189952d-4818-49a6-b83d-64fe35e28625)

**Comment by ektogamat** [comment]
We just need to remove the encoding fragment included. It changed I guess. I will do it 👍🏻.

**Comment by kitaedesigns** [comment]
Thanks @ektogamat ! Also, is that something I can easily do on my end?

**Comment by kitaedesigns** [comment]
I updated to three.js r168 and removed encoding fragment from the "#include <encodings fragment> and replaced with the latest "#include <colorspace_fragment>"  in LensFlare.js and now have a significant performance and framerate drop when all Lens Flare effects are lined up even when occluded. 

![image](https://github.com/user-attachments/assets/0d82818a-a366-4c39-a931-f0ff1dd1b832)

**Comment by ektogamat** [comment]
Yeah. I'm about to launch a new version of lens flare that is not relying on a raycaster to calculate the occlusion. The performance is blazing fast. I will probably launch later this month. 

Thank you very much for your reply here

**Comment by ektogamat** [comment]
This is the new version: https://github.com/N8python/nektoFlare

I created it with N8. There is no documentation yet but it is really straightforward using it. Let me know if you got it working.

**Comment by kitaedesigns** [comment]
Awesome! Will try it out @ektogamat and @N8python !
